### PR TITLE
Endpoints

### PR DESCRIPTION
**Pull request by mjconnor**

### CODE REVIEW

File: templates/endpoints/models/meta-llama--Llama-2-13b-chat-hf.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      do_sample: true
+      temperature: 0.4
+      top_p: 1.0
+      ignore_eos_token: false
+    prompt_format:
+      system: "[INST] <<SYS>>\n{instruction}\n<</SYS>>\n\n"
+      assistant: " {instruction} </s><s> [INST] "
+      trailing_assistant: " "
+      user: "{instruction} [/INST]"
+      default_system_message: "You are a helpful, respectful and honest assistant. Always answer as helpfully as possible, while being safe.  Your answers should not include any harmful, unethical, racist, sexist, toxic, dangerous, or illegal content. Please ensure that your responses are socially unbiased and positive in nature.\n\nIf a question does not make any sense, or is not factually coherent, explain why instead of answering something not correct. If you don't know the answer to a question, please don't share false information."
```
Comment (gvspraveen) [inline templates/endpoints/models/meta-llama--Llama-2-13b-chat-hf.yaml:39]: lets keep default message consistent for both models.

File: templates/endpoints/service.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+name: endpoints
```
Comment (gvspraveen) [inline templates/endpoints/service.yaml:1]: lets wait till we test service roll out with new image tomorrow.

File: templates/endpoints/models/meta-llama--Llama-2-13b-chat-hf.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
+# This model requires HUGGING_FACE_HUB_TOKEN environment variable to be set to a Hugging Face
+# Access Token for an account with permissions to download the model.
+deployment_config:
+  autoscaling_config:
+    min_replicas: 1
+    initial_replicas: 1
+    max_replicas: 1
+    target_num_ongoing_requests_per_replica: 400.0
```
Comment (gvspraveen) [inline templates/endpoints/models/meta-llama--Llama-2-13b-chat-hf.yaml:8]: this should be around 60. 400 seems a lot :D

Lets also limit max_concurrent_queries to 80 or 90

File: templates/endpoints/models/meta-llama--Llama-2-13b-chat-hf.yaml (hunk 1 above)
Comment (mjconnor) [reply templates/endpoints/models/meta-llama--Llama-2-13b-chat-hf.yaml:39]: which one is right @gvspraveen ? the 7b one?

**Review by gvspraveen — APPROVED** [review]

**Pull request merged**